config/jobs/periodic/docker-in-docker/periodic-dind-build.yaml

```diff
@@ -0,0 +1,213 @@
+            exit 1
+          fi
+          wget -O ${PATH_SCRIPTS}/dockerd-entrypoint.sh https://raw.githubusercontent.com/docker-library/docker/master/dockerd-entrypoint.sh 
+          if ! test -f ${PATH_SCRIPTS}/dockerd-entrypoint.sh
+          then
+            echo "The dockerd-entrypoint file was not downloaded." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            exit 1
+          fi
+          chmod a+x ${PATH_SCRIPTS}/*.sh
+
+          # start the dockerd
+          echo "** Dockerd **" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+          # bash ${PATH_SCRIPTS}/dockerd-entrypoint.sh &
+          source ${PATH_SCRIPTS}/dockerd-starting.sh
+
+          if [ -z "$pid" ]
+          then
+            echo "There is no docker daemon." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            exit 1
+          else
+            # get the env file and the dockertest repo and the latest built of containerd if we don't want to build containerd
+            echo "*** COS Bucket ***" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            CONT_NAME=docker_s3_env
+            docker run --env S3_SECRET_AUTH --env PATH_LOG_PROWJOB -d -v /workspace:/workspace --privileged --name $CONT_NAME debian:bullseye /bin/bash -c "${PATH_SCRIPTS}/get_env.sh"
+            status_code="$(docker container wait $CONT_NAME)"
+            echo $status_code 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+
+            if [[ ${status_code} -ne 0 ]]
+            then
+              echo "The docker to get the env.list and the dockertest has failed." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              docker logs $CONT_NAME 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              exit 1
+            fi
+
+            ls /workspace 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            
+            if [[ -f env.list ]]
+            then
+              # check there are 3 env variables in env.list
+              cat env.list 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              if grep -Fq "DOCKER_VERS" env.list && grep -Fq "CONTAINERD_VERS" env.list && grep -Fq "PACKAGING_REF" env.list
+              then 
+                echo "DOCKER_VERS, CONTAINERD_VERS, PACKAGING_REF are in env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              else 
+                echo "DOCKER_VERS, CONTAINERD_VERS and/or PACKAGING_REF are not in env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+                exit 1
+              fi
+              set -o allexport
+              source env.list
+            else
+              echo "There is no env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            fi
+            
+            # generate the env-distrib.list
+            echo "*** * env-distrib.list * ***" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            mkdir docker-ce-packaging
+            pushd docker-ce-packaging
+            git init
+            git remote add origin  https://github.com/docker/docker-ce-packaging.git
+            git fetch --depth 1 origin ${PACKAGING_REF}
```

Comment:
Please see if cloning of `docker-ce-packaging.git` can be done through extra_ref